admin_manual/configuration_user/user_configuration.rst

```diff
@@ -186,13 +186,13 @@ Disable and enable users
 
 .. figure:: ../images/users-actions.png
 
-Sometimes you may want to disable a user without permanently deleting his
+Sometimes you may want to disable a user without permanently deleting their
```

Comment:
Why not use the plural of user in the first place?

Reply:
What do you mean? Could you give an example sentence? 😊

Reply:
```suggestion
Sometimes you may want to disable users without permanently deleting them
```

Reply:
Oh, I see. Well, it didn't really strike me as something to do. If y'all want to change it that way, I don't mind; should still be `their` though, to make sense with the rest of the sentence: `settings and files. The user can be activated any time again, without data-loss.` 😊

Reply:
This change seems totally fine as is. :)